src/AVFoundation/AVContentKeySession.cs

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright 2021 Microsoft Corp
+//
+// Authors:
+//   Alex Soto (alexsoto@microsoft.com)
+//   Rachel Kang (rachelkang@microsoft.com)
+//
+
+#if false
+using System;
+using System.Runtime.InteropServices;
+using CoreGraphics;
+using ObjCRuntime;
+
+namespace AVFoundation {
+    public partial class AVContentKeySession {
+
+        [iOS (14, 5), Mac (11, 3), TV (14, 5), Watch (7,4)]
+        [DllImport (Constants.AVFoundationLibrary)]
+        [return: MarshalAs (UnmanagedType.I1)]
+        static extern /* BOOL */ bool AVSampleBufferAttachContentKey (
+            /* CMSampleBufferRef */ IntPt sbuf,
+            /* AVContentKey */ IntPt contentKey,
+            /* NSError * _Nullable * _Nullable */ out IntPtr outError);
```

Comment:
Nice trick I use, you can do a 'type alias' with the using keyword over the comment, for example:
```csharp
using CMSampleBufferRef =System.IntPtr;
```
Example: https://github.com/xamarin/xamarin-macios/blob/main/src/Network/NWBrowser.cs#L19
Later you can do:
```csharp
static extern /* BOOL */ bool AVSampleBufferAttachContentKey (
            CMSampleBufferRef  sbuf,
```
Much nicer to read :)

Reply:
ooo nice! thanks for the tip :)